robottelo/config/validators.py

```diff
@@ -6,7 +6,15 @@
 
 validators = dict(
     server=[
-        Validator("server.hostname", must_exist=True),
+        Validator("server.hostname", must_exist=False),
+        Validator("server.hostnames", must_exist=True, is_type_of=list),
+        Validator("server.version.release", must_exist=True),
+        Validator("server.version.source", must_exist=True),
+        Validator(
+            "server.xdist_behavior", must_exist=True, is_in=['run-on-one', 'balance', 'on-demand']
+        ),
+        Validator("server.on_demand", must_exist=True, is_type_of=bool),
+        Validator("server.auto_checkin", must_exist=True, is_type_of=bool),
```

Comment:
I think this could be defaulted instead of must_exist?

Reply:
Yeah I could do that. I'd likely set the default to False, but this would need to be updated by CI to True. We can add that additional logic later for setting auto_checkin to False if x% of tests fail in a session.

Reply:
fixed